dojo/importers/reimporter/utils.py

```diff
@@ -213,8 +213,14 @@ def get_or_create_engagement(engagement_id=None, engagement_name=None, product_n
         if not product:
             raise ValueError('no product, unable to create engagement')
 
+        target_start = timezone.now().date()
+        if isinstance(target_end, type(None)):
+            target_end = (timezone.now() + timedelta(days=365)).date()
+        if target_start > target_end:
+            target_end = (timezone.now() + timedelta(days=365)).date()
```

Comment:
Couldn't these be combined in the same if statement?

Reply:
you are right. This happened because I introduced the second if afterwards and changed the target_end value.